src/shell/detect.rs

```diff
@@ -15,6 +16,7 @@ impl ShellKind {
         Some(match name {
             "bash" | "dash" => ShellKind::Bash,
             "fish" => ShellKind::Fish,
+            "xonsh" | "python" => ShellKind::Xonsh,
```

Comment:
Let's hope not too many shells are made with Python 😄

Reply:
I figured we'd let them cross that bridge when they get here :grin: The alternative would probably be to, if python was detected, strip the interpreter and then reevaluate the remaining string as if it were the original command.

Reply:
Yeah we'll cross the bridge when we get to it